### PR TITLE
please, approve the small changes to support custom list, adapter, layout and dialog

### PR DESCRIPTION
**Pull request by sawrus**
Hello, 
please, approve the small changes to support custom list, adapter, layout and dialog.

thank you!.
--
Evgeny

### CODE REVIEW

**Comment by soarcn** [comment]
Hi @sawrus 

Thanks for your PR,  I'm afraid I can't approve this change.

If you want to customize the look&feel of bottomsheet with your own layout, you can simply provide your own header/list_item layout in the style. Check the last example of sample application to know more (R.style.BottomSheet_CustomizedDialog)

**Comment by sawrus** [comment]
So, Could you explain me How Can I dynamically add the new fragment ?

**Comment by soarcn** [comment]
Hi @sawrus 
I create this library for creating a simple Model Bottomsheet component easy. In this case, it could be used for replacing menu/context menu/ list dialog . it should provide constant look and feel in whole app. 

In case you need to dynamically change the whole UI of bottomsheet, you are actually looking for a Persistent Bottom Sheets, and you'd better turn to slidinguplayout or flipboard's bottomsheet. 

Check google's doc for those 2 models, https://www.google.com/design/spec/components/bottom-sheets.html

In conclusion, this library only supports customize header layout and list layout.
